.gitignore

```diff
@@ -40,3 +40,5 @@ TestResult.xml
 *.err
 GlobalAssemblyInfo.cs
 CommonAssemblyInfo.cs
+All Tests in CorrugatedIron.dcvr.settings
```

Comment:
"does ignore the file" - so `All Tests in CorrugatedIron.dcvr.settings` is an actual file? Can you change the line to this?

```
*.dcvr.settings
```